sets/sets.gobra

```diff
@@ -0,0 +1,529 @@
+/*
+  This file is part of gobra-libs which is released under the MIT license.
+  See LICENSE or go to https://github.com/viperproject/gobra-libs/blob/main/LICENSE
+  for full license details.
+
+  This file is inspired by the following content:
+  - dafny-lang/libraries: https://github.com/dafny-lang/libraries/blob/master/src/Collections/Sets/Sets.dfy
+  - verus-lang/verus: https://github.com/verus-lang/verus/blob/main/source/pervasive/set_lib.rs
+  - why3: https://why3.lri.fr/stdlib/set.html
+  - viperproject/silicon: https://github.com/viperproject/silicon/blob/master/src/main/resources/dafny_axioms/sets.vpr
+
+*/
+
+// This package defines lemmas for sets commonly used in specifications.
+package sets
+
+// ##(-I ../)
+import utils "utils"
+
+// QUES Do we want to have comments for every function? In some cases it is trivial, in other
+// cases it feels like the natural language version is less understandable than the formal specs.
```

Comment:
I will leave it up to you to decide, and I will let you know if I think some description is missing